### DISCUSSION TITLE
Multiple "Continuous build" Releases on our "Releases" page again

### DISCUSSION THREAD

**Discussion by justinclift** [sqlitebrowser/sqlitebrowser#3371] [General]
Looking at our [Releases](https://github.com/sqlitebrowser/sqlitebrowser/releases) page just now, we appear to have several "Continuous build" releases there:

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/406299/9eed4a51-f14e-459a-9eac-2814f1d0fdb0)

One of those has the new "Nightly Build" files created using GitHub Actions, whereas all of them have an AppImage in them.

Does anyone know how easy it would be to stop this duplication and also have things named a bit better?

**Comment by FriedrichFroebel** [comment]
The releases you mention are drafts and not visible publicly.

**Reply by justinclift** [reply]
Oh, I hadn't noticed the "Draft" tag on them, thanks. :smile:

**Comment by mgrojo** [comment]
The draft releases seem leftovers from a previous continuous build system, because we have only two, one from Jun and another from May. The current continuous release seems to be updating a single entry with the Pre-release tag. I'd delete the drafts just for cleanliness, even if they're hidden to the public.

**Reply by justinclift** [reply]
Thanks.  Just deleted them now. :smile:

**Comment by lucydodo** [comment]
I'm preparing some improvements for the current CI workflow soon,
and I'll also look into a way to prevent creating unnecessary Draft releases.

**Comment by lucydodo** [comment]
A commit with improvements for this has just been pushed - no more unnecessary draft release.

**Reply by justinclift** [reply]
Awesome! :smile:
